protocol-tests/src/main/java/fixtures/constants/AutoRestSwaggerConstantServiceAsyncClient.java

```diff
@@ -120,7 +118,7 @@ public Mono<Response<Void>> putNoModelAsStringNoRequiredOneValueDefaultWithRespo
     /**
      * Puts constants to the testserver.
      * 
-     * @param input The input parameter. Allowed values: "value1", "value2".
+     * @param input . Allowed values: "value1", "value2".
```

Comment:
We probably want "The input parameter. Allowed values: "value1", "value2"." here.

The ". Allowed .." likely added by codegen. Swagger prorbably have no description on this param.

Reply:
Thanks for the catch! fixing.

Reply:
Fixed in 7b1bc2e94d281c8e105a85a6d508c4c063d5d777